frontend_app_logbook/src/component/Logbook.jsx

```diff
@@ -95,6 +100,25 @@ export class Logbook extends React.Component {
     }
   }
 
+  mapedEntriesWithOldExpand = (entries) => {
```

Comment:
recheck this function, it seems odd.
Check naming, the forEach instead on map and the filter instead of find